docs/src/main/sphinx/admin/fault-tolerant-execution.md

```diff
@@ -449,7 +461,11 @@ the property may be configured for:
   - AWS S3, GCS
 * - `exchange.s3.endpoint`
   - S3 storage endpoint server if using an S3-compatible storage system that
-    is not AWS. If using AWS S3, this can be ignored. If using GCS, set it
+    is not AWS. If using AWS S3, this can be ignored unless TLS is required 
+    by an AWS bucket policy. If TLS is required, then this property can be 
+    set to an https endpoint such as ``https://s3.us-east-1.amazonaws.com``. 
+    Note that TLS is redundant due to {ref}`internal encryption <fte-encryption>`. 
```

Comment:
```suggestion
    Note that TLS is redundant due to [internal encryption](fte-encryption). 
```

Also .. maybe automatic encryption ?

Reply:
I debated automatic vs internal, so i'll count that as two votes for "automatic"